parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -413,7 +413,10 @@ check_array_size() {
 
 capture_benchmark() {
   set +x
-  benchmarks+=($1)
```

Comment:
for future reference, let's avoid using parameters directly within logic, e.g. always assign to a local variable to give the parameter a meaningful name before using elsewhere within the function/script.

Reply:
Will do!